cloudos_cli/related_analyses/related_analyses.py

```diff
@@ -0,0 +1,238 @@
+        json.dump(data, f, indent=4)
+
+
+def save_as_stdout(data, j_workdir_parent, cloudos_url="https://cloudos.lifebit.ai"):
+    """Display related analyses in a formatted table with pagination.
+    
+    Parameters
+    ----------
+    data : dict
+        Dictionary where keys are job IDs and values are dictionaries
+        containing job details (status, name, user_name, user_surname,
+        _id, createdAt, runTime, computeCostSpent).
+    """
+    console = Console(markup=True)
+
+    # Helper function to format timestamp
+    def format_timestamp(timestamp_str):
+        if not timestamp_str:
+            return "N/A"
+        try:
+            dt = datetime.fromisoformat(timestamp_str.replace('Z', '+00:00'))
+            return dt.strftime('%Y-%m-%d %H:%M:%S')
+        except (ValueError, AttributeError):
+            return timestamp_str
+
+    # Helper function to format runtime
+    def format_runtime(runtime_seconds):
+        if runtime_seconds is None:
+            return "N/A"
+        try:
+            total_seconds = int(runtime_seconds)
+            hours = total_seconds // 3600
+            minutes = (total_seconds % 3600) // 60
+            seconds = total_seconds % 60
+            if hours > 0:
+                return f"{hours}h {minutes}m {seconds}s"
+            elif minutes > 0:
+                return f"{minutes}m {seconds}s"
+            else:
+                return f"{seconds}s"
+        except (ValueError, TypeError):
+            return "N/A"
+
+    # Helper function to format cost
+    def format_cost(cost):
+        if cost is None or cost == "":
+            return "N/A"
+        try:
+            # Cost is stored in cents, divide by 100 to get dollars
+            return f"${float(cost) / 100:.4f}"
+        except (ValueError, TypeError):
+            return "N/A"
+
+    # Prepare all rows data
+    rows = []
+
+    for job_id, job_info in data.items():
+        status = job_info.get('status', 'N/A')
+        name = job_info.get('name', 'N/A')
+        user_name = job_info.get('user_name', '')
+        user_surname = job_info.get('user_surname', '')
+        owner = f"{user_name} {user_surname}".strip() if user_name or user_surname else "N/A"
+        job_id_display = job_info.get('_id', job_id)
+        submit_time = format_timestamp(job_info.get('createdAt'))
+        run_time = format_runtime(job_info.get('runTime'))
+        cost = job_info.get('computeCostSpent')
+        total_cost_formatted = format_cost(cost)
+
+        # Add hyperlink to job_id_display
+        job_url = f"{cloudos_url}/app/advanced-analytics/analyses/{job_id_display}"
+        job_id_with_link = f"[link={job_url}]{job_id_display}[/link]"
+
+        rows.append([
+            status,
+            name,
+            owner,
+            job_id_with_link,
+            submit_time,
+            run_time,
+            total_cost_formatted
+        ])
+
+    # Pagination setup
+    limit = 10  # Display 10 rows per page
```

Comment:
I though you were going to parametrise it. It can be done in next iteration when we also add the --all option